test/backup.ts

```diff
@@ -328,6 +339,49 @@ describe('Bigtable/Backup', () => {
     });
   });
 
+  describe('getIamPolicy', () => {
+    it('should provide the proper request options', done => {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      backup.bigtable.request = (config: any) => {
```

Comment:
I don't think we should do this extensive of testing. The method itself does not call this method or set the strings and values being tested here. Same for the error handling and callback executing, etc as those are behaviors of the Table class. Instead, we should stub the Table.prototype methods, and be sure they are called correctly.

Reply:
Done!
Thanks

Reply:
Nice, thank you!